Incompatibility with ember-cli 3.x

@dfreeman would you mind taking a quick peek at [this](https://github.com/stonecircle/ember-cli-notifications/issues/200)? This addon is using `ember-css-modules` as the docs specify (as far as I can tell) and it works fine with ember-cli v2.18.2 and prior but breaks with v3.0.0-beta.1 and newer. The `import` statement pointing to the `.css` file seems to cause a `Cannot find module` runtime error as if it's looking for `.js` instead and isn't finding it.

Although `ember-css-modules` tests are passing, if you update the ember-cli version in the package.json to `~3.0.0-beta.2` it blows up:
<summary>
44 fail, 77 pass
<details>

```
$ ember test
Could not start watchman
Visit https://ember-cli.com/user-guide/#watchman for more info.
cleaning up...
Built project successfully. Stored in "/home/jquant/tmp/ember-css-modules/tmp/class-tests_dist-dLSlZFFA.tmp".
ok 1 Chrome 64.0 - ESLint | addon: addon/extensions.js
    ---
        Log: |
            { type: 'warn',
              text: '\'DEPRECATION: `setResolver` should be imported from `@ember/test-helpers`, but was imported from `ember-qunit` [deprecation id: ember-qunit.deprecated-reexports.setResolver]\\n        at logDeprecationStackTrace (http://localhost:7357/assets/vendor.js:24121:21)\\n        at HANDLERS.(anonymous function) (http://localhost:7357/assets/vendor.js:24343:9)\\n        at raiseOnDeprecation (http://localhost:7357/assets/vendor.js:24151:14)\\n        at HANDLERS.(anonymous function) (http://localhost:7357/assets/vendor.js:24343:9)\\n        at invoke (http://localhost:7357/assets/vendor.js:24355:9)\\n        at Object.deprecate (http://localhost:7357/assets/vendor.js:24209:24)\\n        at setResolver (http://localhost:7357/assets/test-support.js:24460:32)\\n        at Module.callback (http://localhost:7357/assets/tests.js:1027:31)\\n        at Module.exports (http://localhost:7357/assets/vendor.js:109:32)\'\n' }
    ...
ok 2 Chrome 64.0 - ESLint | addon: addon/helpers/local-class.js
ok 3 Chrome 64.0 - ESLint | addon: addon/mixins/component-lookup-mixin.js
ok 4 Chrome 64.0 - ESLint | addon: addon/mixins/component-mixin.js
ok 5 Chrome 64.0 - ESLint | addon: addon/mixins/controller-mixin.js
ok 6 Chrome 64.0 - ESLint | addon: addon/mixins/resolver-mixin.js
ok 7 Chrome 64.0 - ESLint | app: app/helpers/local-class.js
ok 8 Chrome 64.0 - Acceptance | style imports: component-with-relative-imports
ok 9 Chrome 64.0 - Acceptance | style imports: component-with-absolute-imports
ok 10 Chrome 64.0 - Acceptance | style imports: component-with-virtual-imports
not ok 11 Chrome 64.0 - Global error: Uncaught Error: Could not find module `dummy-addon/styles/components/testing/addon-component-with-relative-imports` imported from `dummy-addon/components/testing/addon-component-with-relative-imports` at http://localhost:7357/assets/vendor.js, line 53761
    ---
        Log: |
            { type: 'error',
              text: 'Uncaught Error: Could not find module `dummy-addon/styles/components/testing/addon-component-with-relative-imports` imported from `dummy-addon/components/testing/addon-component-with-relative-imports` at http://localhost:7357/assets/vendor.js, line 53761\n' }
    ...
not ok 12 Chrome 64.0 - Acceptance | style imports: addon-component-with-relative-imports
    ---
        actual: >
            null
        stack: >
            http://localhost:7357/assets/vendor.js:53761
        message: >
            Uncaught Error: Could not find module `dummy-addon/styles/components/testing/addon-component-with-relative-imports` imported from `dummy-addon/components/testing/addon-component-with-relative-imports`
        Log: |
    ...
not ok 13 Chrome 64.0 - Global error: Uncaught Error: Could not find module `dummy-addon/styles/components/testing/addon-component-with-absolute-imports` imported from `dummy-addon/components/testing/addon-component-with-absolute-imports` at http://localhost:7357/assets/vendor.js, line 53761
    ---
        Log: |
            { type: 'error',
              text: 'Uncaught Error: Could not find module `dummy-addon/styles/components/testing/addon-component-with-absolute-imports` imported from `dummy-addon/components/testing/addon-component-with-absolute-imports` at http://localhost:7357/assets/vendor.js, line 53761\n' }
    ...
not ok 14 Chrome 64.0 - Acceptance | style imports: addon-component-with-absolute-imports
    ---
        actual: >
            null
        stack: >
            http://localhost:7357/assets/vendor.js:53761
        message: >
            Uncaught Error: Could not find module `dummy-addon/styles/components/testing/addon-component-with-absolute-imports` imported from `dummy-addon/components/testing/addon-component-with-absolute-imports`
        Log: |
    ...
not ok 15 Chrome 64.0 - Global error: Uncaught Error: Could not find module `dummy-addon/styles/components/testing/addon-component-with-virtual-imports` imported from `dummy-addon/components/testing/addon-component-with-virtual-imports` at http://localhost:7357/assets/vendor.js, line 53761
    ---
        Log: |
            { type: 'error',
              text: 'Uncaught Error: Could not find module `dummy-addon/styles/components/testing/addon-component-with-virtual-imports` imported from `dummy-addon/components/testing/addon-component-with-virtual-imports` at http://localhost:7357/assets/vendor.js, line 53761\n' }
    ...
not ok 16 Chrome 64.0 - Acceptance | style imports: addon-component-with-virtual-imports
    ---
        actual: >
            null
        stack: >
            http://localhost:7357/assets/vendor.js:53761
        message: >
            Uncaught Error: Could not find module `dummy-addon/styles/components/testing/addon-component-with-virtual-imports` imported from `dummy-addon/components/testing/addon-component-with-virtual-imports`
        Log: |
    ...
ok 17 Chrome 64.0 - Acceptance | component styles lookup: root-pod-component
ok 18 Chrome 64.0 - Acceptance | component styles lookup: root-pod-template-only-component
ok 19 Chrome 64.0 - Acceptance | component styles lookup: classic-component
ok 20 Chrome 64.0 - Acceptance | component styles lookup: classic-template-only-component
ok 21 Chrome 64.0 - Acceptance | component styles lookup: pod-component
ok 22 Chrome 64.0 - Acceptance | component styles lookup: pod-template-only-component
not ok 23 Chrome 64.0 - Global error: Uncaught Error: Could not find module `dummy-addon/styles/components/testing/addon-component` imported from `dummy-addon/components/testing/addon-component` at http://localhost:7357/assets/vendor.js, line 53761
    ---
        Log: |
            { type: 'error',
              text: 'Uncaught Error: Could not find module `dummy-addon/styles/components/testing/addon-component` imported from `dummy-addon/components/testing/addon-component` at http://localhost:7357/assets/vendor.js, line 53761\n' }
    ...
not ok 24 Chrome 64.0 - Acceptance | component styles lookup: addon-component
    ---
        actual: >
            null
        stack: >
            http://localhost:7357/assets/vendor.js:53761
        message: >
            Uncaught Error: Could not find module `dummy-addon/styles/components/testing/addon-component` imported from `dummy-addon/components/testing/addon-component`
        Log: |
    ...
ok 25 Chrome 64.0 - Acceptance | component styles lookup: pod-route/nested-pod-component
ok 26 Chrome 64.0 - Acceptance | component styles lookup: pod-route/nested-pod-template-only-component
ok 27 Chrome 64.0 - Acceptance | component styles lookup: component-with-dynamic-local-class
ok 28 Chrome 64.0 - Acceptance | component styles lookup: component-with-global-class-and-dynamic-local-class
ok 29 Chrome 64.0 - Acceptance | component styles lookup: component-with-local-class-helper
ok 30 Chrome 64.0 - Acceptance | component styles lookup: component-with-global-class-composition
ok 31 Chrome 64.0 - Acceptance | component styles lookup: component-with-container-class
ok 32 Chrome 64.0 - Acceptance | component styles lookup: component-with-addon-value
ok 33 Chrome 64.0 - Acceptance | component styles lookup: component-with-addon-composition
not ok 34 Chrome 64.0 - Global error: Uncaught Error: Could not find module `dummy-sass-addon/styles/components/testing/sass-addon-component` imported from `dummy-sass-addon/components/testing/sass-addon-component` at http://localhost:7357/assets/vendor.js, line 53761
    ---
        Log: |
            { type: 'error',
              text: 'Uncaught Error: Could not find module `dummy-sass-addon/styles/components/testing/sass-addon-component` imported from `dummy-sass-addon/components/testing/sass-addon-component` at http://localhost:7357/assets/vendor.js, line 53761\n' }
    ...
not ok 35 Chrome 64.0 - Acceptance | component styles lookup: sass-addon-component
    ---
        actual: >
            null
        stack: >
            http://localhost:7357/assets/vendor.js:53761
        message: >
            Uncaught Error: Could not find module `dummy-sass-addon/styles/components/testing/sass-addon-component` imported from `dummy-sass-addon/components/testing/sass-addon-component`
        Log: |
    ...
not ok 36 Chrome 64.0 - Global error: Uncaught Error: Could not find module `dummy-less-addon/styles/components/testing/less-addon-component` imported from `dummy-less-addon/components/testing/less-addon-component` at http://localhost:7357/assets/vendor.js, line 53761
    ---
        Log: |
            { type: 'error',
              text: 'Uncaught Error: Could not find module `dummy-less-addon/styles/components/testing/less-addon-component` imported from `dummy-less-addon/components/testing/less-addon-component` at http://localhost:7357/assets/vendor.js, line 53761\n' }
    ...
not ok 37 Chrome 64.0 - Acceptance | component styles lookup: less-addon-component
    ---
        actual: >
            null
        stack: >
            http://localhost:7357/assets/vendor.js:53761
        message: >
            Uncaught Error: Could not find module `dummy-less-addon/styles/components/testing/less-addon-component` imported from `dummy-less-addon/components/testing/less-addon-component`
        Log: |
    ...
ok 38 Chrome 64.0 - Acceptance | component styles lookup: classic-route
ok 39 Chrome 64.0 - Acceptance | component styles lookup: classic-template-only-route
ok 40 Chrome 64.0 - Acceptance | component styles lookup: pod-route
ok 41 Chrome 64.0 - Acceptance | component styles lookup: pod-template-only-route
ok 42 Chrome 64.0 - ESLint | app: app.js
ok 43 Chrome 64.0 - ESLint | app: components/testing/classic-component.js
ok 44 Chrome 64.0 - ESLint | app: components/testing/pod-component/component.js
ok 45 Chrome 64.0 - ESLint | app: controllers/testing/classic-route.js
ok 46 Chrome 64.0 - ESLint | app: helpers/test-helper.js
ok 47 Chrome 64.0 - ESLint | app: resolver.js
ok 48 Chrome 64.0 - ESLint | app: router.js
ok 49 Chrome 64.0 - ESLint | app: testing/pod-route/controller.js
ok 50 Chrome 64.0 - ESLint | app: testing/pod-route/nested-pod-component/component.js
ok 51 Chrome 64.0 - ESLint | app: testing/root-pod-component/component.js
not ok 52 Chrome 64.0 - Integration | Changing local classes: changing a dynamic class value works
    ---
        expected: >
            global --foo --bar
        stack: >
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:306:12)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
                at begin (http://localhost:7357/assets/test-support.js:5689:20)
        Log: |
    ...
ok 53 Chrome 64.0 - Integration | "postprocess" Plugins: executes postprocess plugins
not ok 54 Chrome 64.0 - Integration | Helpers still function: helpers work correctly
    ---
        actual: >
            ember-css-modules
        expected: >
            foo, bar
        stack: >
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:353:12)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
                at begin (http://localhost:7357/assets/test-support.js:5689:20)
        Log: |
    ...
not ok 55 Chrome 64.0 - Integration | Helper | local-class: it works as a mustache
    ---
        actual: >
            ember-css-modules
        expected: >
            --foo-value
        stack: >
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:370:12)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
                at begin (http://localhost:7357/assets/test-support.js:5689:20)
        Log: |
    ...
not ok 56 Chrome 64.0 - Integration | Helper | local-class: it works as a positional
    ---
        actual: >
            ember-css-modules
        expected: >
            --foo-value
        stack: >
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:388:12)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
                at begin (http://localhost:7357/assets/test-support.js:5689:20)
        Log: |
    ...
not ok 57 Chrome 64.0 - Integration | Helper | local-class: it works as a named parameter
    ---
        actual: >
            ember-css-modules
        expected: >
            --foo-value
        stack: >
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:409:12)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
                at begin (http://localhost:7357/assets/test-support.js:5689:20)
        Log: |
    ...
ok 58 Chrome 64.0 - Integration | Mixin | component mixin: it exposes a computed __styles__ property
not ok 59 Chrome 64.0 - Integration | Mixin | component mixin: it honors a configured localClassName
    ---
        actual: >
            false
        expected: >
            true
        stack: >
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:454:12)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
                at begin (http://localhost:7357/assets/test-support.js:5689:20)
        message: >
            failed, expected argument to be truthy, was: false
        Log: |
    ...
not ok 60 Chrome 64.0 - Integration | Mixin | component mixin: it honors a configured simple localClassNameBinding
    ---
        actual: >
            false
        expected: >
            true
        stack: >
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:481:12)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
                at begin (http://localhost:7357/assets/test-support.js:5689:20)
        message: >
            failed, expected argument to be truthy, was: false
        Log: |
    ...
not ok 61 Chrome 64.0 - Integration | Mixin | component mixin: it honors a configured mapped localClassNameBinding
    ---
        actual: >
            false
        expected: >
            true
        stack: >
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:513:12)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
                at begin (http://localhost:7357/assets/test-support.js:5689:20)
        message: >
            failed, expected argument to be truthy, was: false
        Log: |
    ...
not ok 62 Chrome 64.0 - Integration | Mixin | component mixin: it honors a configured mapped localClassNameBinding with an inverse
    ---
        actual: >
            false
        expected: >
            true
        stack: >
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:547:12)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
                at begin (http://localhost:7357/assets/test-support.js:5689:20)
        message: >
            failed, expected argument to be truthy, was: false
        Log: |
    ...
not ok 63 Chrome 64.0 - Integration | Mixin | component mixin: it supports localClassNames with composition
    ---
        actual: >
            false
        expected: >
            true
        stack: >
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:576:12)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
                at begin (http://localhost:7357/assets/test-support.js:5689:20)
        message: >
            failed, expected argument to be truthy, was: false
        Log: |
    ...
not ok 64 Chrome 64.0 - Integration | Mixin | component mixin: it supports localClassNameBindings with composition in the positive class
    ---
        actual: >
            false
        expected: >
            true
        stack: >
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:604:12)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
                at begin (http://localhost:7357/assets/test-support.js:5689:20)
        message: >
            failed, expected argument to be truthy, was: false
        Log: |
    ...
not ok 65 Chrome 64.0 - Integration | Mixin | component mixin: it supports localClassNameBindings with composition in the negative class
    ---
        actual: >
            false
        expected: >
            true
        stack: >
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:639:12)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
                at begin (http://localhost:7357/assets/test-support.js:5689:20)
        message: >
            failed, expected argument to be truthy, was: false
        Log: |
    ...
not ok 66 Chrome 64.0 - Integration | Mixin | component mixin: it honors a configured mapped localClassNameBinding string
    ---
        actual: >
            false
        expected: >
            true
        stack: >
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:674:12)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
                at begin (http://localhost:7357/assets/test-support.js:5689:20)
        message: >
            failed, expected argument to be truthy, was: false
        Log: |
    ...
ok 67 Chrome 64.0 - Integration | Mixin | controller mixin: it exposes a computed __styles__ property
not ok 68 Chrome 64.0 - Integration | Template AST Plugin: local-class helper with no source specified
    ---
        actual: >
            0
        expected: >
            true
        stack: >
                at assertTransforms (http://localhost:7357/assets/tests.js:1000:16)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:1014:9)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
        message: >
            Expected output contains verification selector
        Log: |
    ...
not ok 69 Chrome 64.0 - Integration | Template AST Plugin: local-class helper with a source specified
    ---
        actual: >
            0
        expected: >
            true
        stack: >
                at assertTransforms (http://localhost:7357/assets/tests.js:1000:16)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:1014:9)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
        message: >
            Expected output contains verification selector
        Log: |
    ...
not ok 70 Chrome 64.0 - Integration | Template AST Plugin: creating a class attribute
    ---
        actual: >
            0
        expected: >
            true
        stack: >
                at assertTransforms (http://localhost:7357/assets/tests.js:1000:16)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:1014:9)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
        message: >
            Expected output contains verification selector
        Log: |
    ...
not ok 71 Chrome 64.0 - Integration | Template AST Plugin: creating a class attribute with multiple classes
    ---
        actual: >
            0
        expected: >
            true
        stack: >
                at assertTransforms (http://localhost:7357/assets/tests.js:1000:16)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:1014:9)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
        message: >
            Expected output contains verification selector
        Log: |
    ...
not ok 72 Chrome 64.0 - Integration | Template AST Plugin: appending to a class attribute
    ---
        actual: >
            0
        expected: >
            true
        stack: >
                at assertTransforms (http://localhost:7357/assets/tests.js:1000:16)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:1014:9)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
        message: >
            Expected output contains verification selector
        Log: |
    ...
not ok 73 Chrome 64.0 - Integration | Template AST Plugin: appending to a class attribute with multiple classes
    ---
        actual: >
            0
        expected: >
            true
        stack: >
                at assertTransforms (http://localhost:7357/assets/tests.js:1000:16)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:1014:9)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
        message: >
            Expected output contains verification selector
        Log: |
    ...
not ok 74 Chrome 64.0 - Integration | Template AST Plugin: appending to an unquoted string literal class attribute
    ---
        actual: >
            0
        expected: >
            true
        stack: >
                at assertTransforms (http://localhost:7357/assets/tests.js:1000:16)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:1014:9)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
        message: >
            Expected output contains verification selector
        Log: |
    ...
not ok 75 Chrome 64.0 - Integration | Template AST Plugin: appending to an unquoted string literal class attribute with multiple classes
    ---
        actual: >
            0
        expected: >
            true
        stack: >
                at assertTransforms (http://localhost:7357/assets/tests.js:1000:16)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:1014:9)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
        message: >
            Expected output contains verification selector
        Log: |
    ...
not ok 76 Chrome 64.0 - Integration | Template AST Plugin: appending a ConcatStatement class
    ---
        actual: >
            0
        expected: >
            true
        stack: >
                at assertTransforms (http://localhost:7357/assets/tests.js:1000:16)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:1014:9)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
        message: >
            Expected output contains verification selector
        Log: |
    ...
not ok 77 Chrome 64.0 - Integration | Template AST Plugin: appending a static class with a ConcatStatement local-class
    ---
        actual: >
            0
        expected: >
            true
        stack: >
                at assertTransforms (http://localhost:7357/assets/tests.js:1000:16)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:1014:9)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
        message: >
            Expected output contains verification selector
        Log: |
    ...
not ok 78 Chrome 64.0 - Integration | Template AST Plugin: creating a class attribute with dynamic local-class value
    ---
        actual: >
            0
        expected: >
            true
        stack: >
                at assertTransforms (http://localhost:7357/assets/tests.js:1000:16)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:1014:9)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
        message: >
            Expected output contains verification selector
        Log: |
    ...
not ok 79 Chrome 64.0 - Integration | Template AST Plugin: creating a class attribute with mixed local-class value
    ---
        actual: >
            0
        expected: >
            true
        stack: >
                at assertTransforms (http://localhost:7357/assets/tests.js:1000:16)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:1014:9)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
        message: >
            Expected output contains verification selector
        Log: |
    ...
not ok 80 Chrome 64.0 - Integration | Template AST Plugin: appending a class attribute with dynamic local-class value
    ---
        actual: >
            0
        expected: >
            true
        stack: >
                at assertTransforms (http://localhost:7357/assets/tests.js:1000:16)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:1014:9)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
        message: >
            Expected output contains verification selector
        Log: |
    ...
not ok 81 Chrome 64.0 - Integration | Template AST Plugin: appending a class attribute with mixed local-class value
    ---
        actual: >
            0
        expected: >
            true
        stack: >
                at assertTransforms (http://localhost:7357/assets/tests.js:1000:16)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:1014:9)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
        message: >
            Expected output contains verification selector
        Log: |
    ...
not ok 82 Chrome 64.0 - Integration | Template AST Plugin: appending a path class attribute
    ---
        actual: >
            0
        expected: >
            true
        stack: >
                at assertTransforms (http://localhost:7357/assets/tests.js:1000:16)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:1014:9)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
        message: >
            Expected output contains verification selector
        Log: |
    ...
not ok 83 Chrome 64.0 - Integration | Template AST Plugin: appending a helper class attribute
    ---
        actual: >
            0
        expected: >
            true
        stack: >
                at assertTransforms (http://localhost:7357/assets/tests.js:1000:16)
                at Object.<anonymous> (http://localhost:7357/assets/tests.js:1014:9)
                at runTest (http://localhost:7357/assets/test-support.js:4108:30)
                at Test.run (http://localhost:7357/assets/test-support.js:4094:6)
                at http://localhost:7357/assets/test-support.js:4300:12
                at Object.advance (http://localhost:7357/assets/test-support.js:3746:26)
        message: >
            Expected output contains verification selector
        Log: |
    ...
ok 84 Chrome 64.0 - ESLint | tests: acceptance/imports-test.js
ok 85 Chrome 64.0 - ESLint | tests: acceptance/styles-lookup-test.js
ok 86 Chrome 64.0 - ESLint | tests: helpers/destroy-app.js
ok 87 Chrome 64.0 - ESLint | tests: helpers/module-for-acceptance.js
ok 88 Chrome 64.0 - ESLint | tests: helpers/resolver.js
ok 89 Chrome 64.0 - ESLint | tests: helpers/start-app.js
ok 90 Chrome 64.0 - ESLint | tests: integration/changing-dynamic-class-test.js
ok 91 Chrome 64.0 - ESLint | tests: integration/component-using-postprocess-plugin-test.js
ok 92 Chrome 64.0 - ESLint | tests: integration/helper-test.js
ok 93 Chrome 64.0 - ESLint | tests: integration/local-class-test.js
ok 94 Chrome 64.0 - ESLint | tests: integration/mixins/component-mixin-test.js
ok 95 Chrome 64.0 - ESLint | tests: integration/mixins/controller-mixin-test.js
ok 96 Chrome 64.0 - ESLint | tests: integration/template-ast-plugin-test.js
ok 97 Chrome 64.0 - ESLint | tests: test-helper.js
ok 98 Chrome 64.0 - ESLint | tests: unit/custom-name-scoping-test.js
ok 99 Chrome 64.0 - ESLint | tests: unit/helpers/local-class-test.js
ok 100 Chrome 64.0 - ESLint | tests: unit/module-ordering-test.js
not ok 101 Chrome 64.0 - TestLoader Failures: dummy/tests/unit/custom-name-scoping-test: could not be loaded
    ---
        actual: >
            null
        stack: >
            Error: Could not find module `dummy-sass-addon/styles/components/testing/sass-addon-component` imported from `dummy/tests/unit/custom-name-scoping-test`
                at missingModule (http://localhost:7357/assets/vendor.js:250:11)
                at findModule (http://localhost:7357/assets/vendor.js:261:7)
                at Module.findDeps (http://localhost:7357/assets/vendor.js:171:24)
                at findModule (http://localhost:7357/assets/vendor.js:265:11)
                at requireModule (http://localhost:7357/assets/vendor.js:27:15)
                at TestLoader.<anonymous> (http://localhost:7357/assets/test-support.js:24280:11)
                at TestLoader.require (http://localhost:7357/assets/test-support.js:24270:27)
                at TestLoader.loadModules (http://localhost:7357/assets/test-support.js:24262:16)
                at loadTests (http://localhost:7357/assets/test-support.js:24918:22)
                at start (http://localhost:7357/assets/test-support.js:24662:33)
        message: >
            Died on test #1     at TestLoader.moduleLoadFailure (http://localhost:7357/assets/test-support.js:24894:24)
                at TestLoader.<anonymous> (http://localhost:7357/assets/test-support.js:24282:16)
                at TestLoader.require (http://localhost:7357/assets/test-support.js:24270:27)
                at TestLoader.loadModules (http://localhost:7357/assets/test-support.js:24262:16)
                at loadTests (http://localhost:7357/assets/test-support.js:24918:22)
                at start (http://localhost:7357/assets/test-support.js:24662:33)
                at Module.callback (http://localhost:7357/assets/tests.js:1028:28): Could not find module `dummy-sass-addon/styles/components/testing/sass-addon-component` imported from `dummy/tests/unit/custom-name-scoping-test`
        Log: |
    ...
ok 102 Chrome 64.0 - Unit | Helper | local-class: applying local-class
ok 103 Chrome 64.0 - Unit | Helper | local-class: applying multiple local-class
ok 104 Chrome 64.0 - Unit | Helper | local-class: applying to no style map
ok 105 Chrome 64.0 - Unit | Helper | local-class: applying local-class to non-existent map
ok 106 Chrome 64.0 - Unit | Helper | local-class: with no source specified
ok 107 Chrome 64.0 - Unit | Helper | local-class: with an empty source specified
ok 108 Chrome 64.0 - Unit | Helper | local-class: with an undefined local class
ok 109 Chrome 64.0 - Unit | Helper | local-class: with a string source specified
ok 110 Chrome 64.0 - Unit | Module Ordering: modules are ordered according to composition and explicit directives
ok 111 Chrome 64.0 - ember-qunit: Ember.onerror validation: Ember.onerror is functioning properly
ok 112 Chrome 64.0 - Unit | Helper | local-class: applying local-class
ok 113 Chrome 64.0 - Unit | Helper | local-class: applying multiple local-class
ok 114 Chrome 64.0 - Unit | Helper | local-class: applying to no style map
ok 115 Chrome 64.0 - Unit | Helper | local-class: applying local-class to non-existent map
ok 116 Chrome 64.0 - Unit | Helper | local-class: with no source specified
ok 117 Chrome 64.0 - Unit | Helper | local-class: with an empty source specified
ok 118 Chrome 64.0 - Unit | Helper | local-class: with an undefined local class
ok 119 Chrome 64.0 - Unit | Helper | local-class: with a string source specified
not ok 120 Chrome 64.0 - Global error: Uncaught Error: 
Error: Could not find module `dummy-sass-addon/styles/components/testing/sass-addon-component` imported from `dummy/tests/unit/custom-name-scoping-test` at http://localhost:7357/assets/test-support.js, line 24875
    ---
        Log: |
            { type: 'error',
              text: 'Uncaught Error: \nError: Could not find module `dummy-sass-addon/styles/components/testing/sass-addon-component` imported from `dummy/tests/unit/custom-name-scoping-test` at http://localhost:7357/assets/test-support.js, line 24875\n' }
    ...
not ok 121 Chrome 64.0 - Unit | Helper | local-class: global failure
    ---
        actual: >
            null
        stack: >
            http://localhost:7357/assets/test-support.js:24875
        message: >
            Uncaught Error: 
            Error: Could not find module `dummy-sass-addon/styles/components/testing/sass-addon-component` imported from `dummy/tests/unit/custom-name-scoping-test`
        Log: |
    ...

1..121
# tests 121
# pass  77
# skip  0
# fail  44
Testem finished with non-zero exit code. Tests failed.
```

</details>
</summary>

BTW, it would be nice if running `npm install` would trigger `npm run install-all` (or `yarn run install-all`). Otherwise attempting to run any `ember ...` command results in an error. e.g.:
```
Cannot find module 'postcss-scss'
Error: Cannot find module 'postcss-scss'
    at Function.Module._resolveFilename (module.js:538:15)
    at Function.Module._load (module.js:468:25)
    at Module.require (module.js:587:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/jquant/tmp/ember-css-modules/tests/dummy-sass-addon/index.js:14:17)
    at Module._compile (module.js:643:30)
    at Object.Module._extensions..js (module.js:654:10)
    at Module.load (module.js:556:32)
    at tryModuleLoad (module.js:499:12)
    at Function.Module._load (module.js:491:3)
```

--- a/index.js
+++ b/index.js
@@ -126,10 +126,10 @@ module.exports = {
 
   getAddonModulesRoot() {
     // CLI 2.12 stopped exposing addon stuff nested under `modules/`
-    if (this.checker.for('ember-cli', 'npm').satisfies('>= 2.12')) {
-      return '';
-    } else {
+    if (this.checker.for('ember-cli', 'npm').satisfies('< 2.12')) {
       return 'modules/';
+    } else {
+      return '';
     }
   },
 

